snippets/typescript.json

```diff
@@ -160,5 +160,19 @@
       "export class ${Name}Module { }",
       "$0"
     ]
+  },
+  "Angular Directive": {
+    "prefix": "a-directive",
+    "description": "Angular directive snippet",
+    "body": [
+      "import { Directive, ElementRef } from '@angular/core';",
+      "",
+      "@Directive({ selector: '[${feature}]' })",
+      "export class ${Feature}Directive {",
+      "\tconstructor(el: ElementRef) {",
```

Comment:
IMHO, it would be simpler to let the developer choose what to inject in the ctor:
`constructor($1) {`